### PR TITLE
Added a new component 'Labeled Toggle'

### PR DESCRIPTION
**Pull request by ITs-AIKO**
-  Added a new component 'Labeled Toggle':
  > New file: src/showcase/components/toggle/LabeledToggle.tsx

- Added 'Labeled Toggle' to the documentation:
  > Modified: src/showcase/ui-group/ToggleCards.tsx
  > New file: src/app/docs/components/toggle/labeled-toggle/page.mdx

- Updated the credits page:
  > Modified: credits.md

### CODE REVIEW

File: src/showcase/ui-group/ToggleCards.tsx [hunk 1]
```diff
@@ -19,6 +20,11 @@ const data: ToggleCardProps[] = [
     link: '/docs/components/toggle/neubrutalism-toggle',
     component: <NeubrutalismToggle />,
   },
+  {
+    title: 'idk Toggle',
+    link: '/docs/components/toggle/labeled-toggle',
+    component: <LabeledToggle ON />,
+  },
```
Comment (epoll31) [inline src/showcase/ui-group/ToggleCards.tsx:27]: Can you fix the title in the UI Group?

File: credits.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - LineTabs - [Aiko](https://twitter.com/username_aiko)
 - IconTabs - [Aiko](https://twitter.com/username_aiko)
 - Simple Toggle - [Ethan Pollack](https://epoll31.github.io)
+- Labeled Toggle - [Aiko](https://twitter.com/username_aiko)
```
Comment (epoll31) [inline credits.md:9]: Can you move this to the bottom of `credits.md`?

File: src/app/docs/components/toggle/labeled-toggle/page.mdx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const metadata = {
+  title: 'Toggle',
+  description: 'A toggle switch where the handle switches between two states.',
+}
```
Comment (epoll31) [inline src/app/docs/components/toggle/labeled-toggle/page.mdx:4]: Can you update the title and description to fit your new toggle instead of just Toggle?

**Review by epoll31 — CHANGES_REQUESTED** [review]

**Comment by ITs-AIKO** [comment]
@epoll31 , This PL is not the latest. I'll update the PL right now! 
Apologies.

**Pull request closed without merging**